tests/apps/perf/pubsub_subscribe_http/go.mod

```
@@ -1,22 +1,19 @@
-module github.com/dapr/dapr/tests/apps/perf/pubsub_subscribe_http
+module github.com/dapr/dapr/tests/apps/perf/pubsub_susbcribe_http
```

Note this was changed and now there's a typo


```suggestion
module github.com/dapr/dapr/tests/apps/perf/pubsub_subscribe_http
```